toolkit/tools/pkg/imagecustomizerlib/cleanUpImage.go

```diff
@@ -0,0 +1,37 @@
+package imagecustomizerlib
+
+import (
+	"github.com/microsoft/azurelinux/toolkit/tools/internal/logger"
+	"github.com/microsoft/azurelinux/toolkit/tools/internal/safechroot"
+	"github.com/microsoft/azurelinux/toolkit/tools/internal/shell"
+	"github.com/sirupsen/logrus"
+)
+
+func cleanTdnfCache(imageChroot *safechroot.Chroot) error {
+
+	logger.Log.Infof("Cleaning up image")
+
+	// clear tdnf cache
+	tdnfClean := func() error {
```

Comment:
Is the `tdnfClean` function really needed? Why not just put this code directly under `UnsafeRun()`?

Reply:
Initially, in this PR, I was performing multiple clean-up steps, so I wrapped them each in their own functions. But since we are only performing the tdnf cleaning step now, I will put this code under `UnsafeRun()`

Reply:
Updated and placed code under `UnsafeRun()`